gradle.properties

```diff
@@ -21,5 +21,9 @@ android.enableJetifier=false
 kotlin.code.style=official
 
 # KMP Targets
-ios=false
-js=false
\ No newline at end of file
+ios=true
```

Comment:
Might switch back to false, but have Actions always build against this so I know if it breaks.